grades/api.py

```diff
@@ -36,10 +36,14 @@ def _compute_grade_for_fa(user_edx_run_data):
     if user_edx_run_data.certificate is not None:
         run_passed = user_edx_run_data.certificate.status == 'downloadable'
         grade = user_edx_run_data.certificate.grade
+        # the following line should be updated when
+        # we add support for honor enrollments and certificates in the edx-api-client
+        payed_on_edx = user_edx_run_data.certificate.certificate_type in ['honor', 'verified']
```

Comment:
I did not do it on purpose because there is another story to move this in the `edx-api-client` library as soon as we have some more info from people at edX.